packages/api-postgres-plugin/index.js

```diff
@@ -21,5 +22,6 @@ module.exports = {
   get_smart_contracts,
   get_actions,
   get_action_details,
+  get_permission_by_public_key,
```

Comment:
should the api call `get_permission**s**_by_public_key`

Reply:
this function only accept one public key and returning one permission only
we can extend it if you want

Reply:
change made